apps/admin-gui/src/app/shared/components/members-candidates-list/members-candidates-list.component.html

```diff
@@ -9,11 +9,11 @@
       class="w-100"
       mat-table
       matSort
-      matSortActive="fullName"
-      matSortDirection="asc"
+      matSortActive="checkbox"
+      matSortDirection="desc"
       matSortDisableClear>
       <ng-container matColumnDef="checkbox">
-        <th *matHeaderCellDef mat-header-cell class="align-checkbox"></th>
+        <th *matHeaderCellDef mat-header-cell mat-sort-header class="align-checkbox"></th>
```

Comment:
I wanted to display selected users first and this was the best way that came to mind (I’m not that skilled with angular). Any tips on how to do this would be appreciated (or I can just remove the changes as you suggested)